wicket-core/src/main/java/org/apache/wicket/protocol/http/DefaultResourceIsolationPolicy.java

```diff
@@ -0,0 +1,36 @@
+package org.apache.wicket.protocol.http;
+
+import org.apache.wicket.protocol.http.ResourceIsolationPolicy;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class DefaultResourceIsolationPolicy implements ResourceIsolationPolicy {
```

Comment:
let's add a small Javadoc on this class pointing to https://web.dev/fetch-metadata/ (which is the inspiration for this default policy) and let's add ourselves as @author :)